README.rst

```diff
@@ -9,12 +9,19 @@
 
 .. image:: https://github.com/iotivity/iotivity-lite/actions/workflows/unittest.yml/badge.svg
    :target: https://github.com/iotivity/iotivity-lite/actions/workflows/unittest.yml
+   
+.. image:: https://github.com/iotivity/iotivity-lite/actions/workflows/unittest.yml/badge.svg
```

Comment:
This badge is added twice

Reply:
more github actions will be added, i will clean it up when that happens